docs/started/installation.md

```diff
@@ -23,15 +23,15 @@ If you want to use the autogenerated entities on startup set that setting to tru
-| HASS_GEN_ENTITIES       | Defaults to False, set True if you want the autogenerated entities                                                                                                                                                                                                    |
-| HASS_LOG_LEVEL          | Defaults to info, values are (trace, debug, info, warning, error)                                                                                                                                                                                                     |
-| TZ                      | You will need to set container time zone to make the scheduler work properly                                                                                                                                                                                          |
-| HASS_RUN_PROJECT_FOLDER | HASS_RUN_PROJECT_FOLDER=/data use the csproject file you provides in the netdaemon folder instead of the built-in. This is for advanced scenarios where you want to run custom NUGET packages and compile with your own csproj file. Normally you would not use this. |
+| Env                         | Description                                                                                                                                                                                                                                                           |
+| -----------------------     | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| HOMEASSISTANT__HOST         | The host that is running Home Assistant (defaults to `localhost`).                                                                                                                                                                                                    |
+| HOMEASSISTANT__PORT         | The port Home Assistant is running on (default to `8123`).                                                                                                                                                                                                            |
+| HOMEASSISTANT__TOKEN        | A Long Lived Acces Token(LLAT) that NetDaemon can use for the comminication with Home Assistant.                                                                                                                                                                      |
+| NETDAEMON__GENERATEENTITIES | Defaults to False, set True if you want the autogenerated entities                                                                                                                                                                                                    |
```

Comment:
In https://deploy-preview-34--netdaemon.netlify.app/docs/api/api_gen_entities#get-started
There are still references to the old

Reply:
HASS_RUN_PROJECT_FOLDER should be like
NETDAEMON__PROJECTFOLDER or what itś called